TweetMate/src/main/java/com/ybsystem/tweetmate/models/entities/twitter/TwitterStatus.java

```diff
@@ -147,19 +157,20 @@ public TwitterStatus(Status status) {
         this.convertedRelativeTime = createRelativeTime(status);
         this.convertedAbsoluteTime = createAbsoluteTime(status);
         this.convertedVia = createVia(status);
+        this.imageUrls = createImageUrls(status);
         this.isMyTweet = status.getUser().getId() == TweetMateApp.getMyUser().getId();
         this.isPublic = isMyTweet || !status.getUser().isProtected();
         this.isTalk = status.getInReplyToStatusId() != -1;
         this.isDetail = false;
-        this.isMediaTweet = !status.isRetweet() && status.getMediaEntities().length != 0;
+        this.isTwiMedia = (imageUrls.size() > 0 && mediaEntities.length > 0);
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Use isEmpty() to check whether the collection is empty or not. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://192.168.11.19:9000/coding_rules#rule_key=squid%3AS1155)

Reply:
Fixed: https://github.com/ybs32/TweetMate-for-Android/pull/85/commits/0a5013e538213bacb2774f24bfa4ed6959fa720c